Assets/Scripts/Parallax_Script.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BG_Parallax : MonoBehaviour {
```

Comment:
I'm assuming that this script will be attached to the camera?
Also, please comment your code.
What are the commented out lines supposed to mean? Have they been accidentally left in?